test/concurrent.js

```diff
@@ -362,7 +362,7 @@ test('all async - no failure - no bail', function (t) {
 });
 
 test('all async - no failure - bail', function (t) {
-	new Concurrent(
+	return new Concurrent(
```

Comment:
There are a bunch of `t.end();` that needs to be removed. Like https://github.com/avajs/ava/pull/1014/files#diff-adeeb9de91f4da36bd570d73ef0d19a1R391